content/departments/security/security-incident-response-process.md

```diff
@@ -2,6 +2,23 @@
 
 The aim of this document is to define the key steps required as part of responding to a security incident, in line with Sourcegraph’s Security Incident Response Policy.
 
+## Security Incident Lead
+
+The incident lead is responsible for resolving the incident as quickly and safely as possible. They are the DRI coordinating the incident, tasked with driving it to resolution and keeping others informed (if it’s a non-customer-impact issue) or coordinating with the messenger on communication (for customer-impact issues).
+
+The security incident lead should be a member of the security team, and may be the same person who identified and raised the incident. For critical severity incidents, it should be the Head of Security.
+
+The incident lead can change throughout the incident depending on severity. For example, it may start as the first responder and switch to another teammate, or rotate depending on timezone. When changing over, the current and new incident leads should meet and fully handover.
+
+A security incident lead is responsible for:
+
+- Following the Security Incident Checklist.
+- Maintaining the list of action items and work streams, and assigning them to responders.
+- Validating action items have been completed.
+- Providing regular incident updates.
+- Ensuring the right teammates are working on the incident.
+- Handing communication within the team, with the company, and working with a messenger for customer-impacting incidents.
+
 ## Triaging a reported event and declaring an incident
 
 When triaging a security event, follow the documented [security event response process](https://github.com/sourcegraph/security-monitoring/blob/main/docs/security-event-handling.md).
```

Comment:
The following lines could use updating/removing - they refer to our old incident response template that we've replaced with incident.io.

Reply:
Got it, i will create a new PR with those changes just so that i can keep them separated for review, thanks!